types/node/globals.d.ts

```diff
@@ -130,263 +91,104 @@ interface NodeDOMExceptionConstructor {
 // #endregion DOMException
 
 declare global {
-    // Declare "static" methods in Error
-    interface ErrorConstructor {
-        /** Create .stack property on a target object */
-        captureStackTrace(targetObject: object, constructorOpt?: Function): void;
-
-        /**
-         * Optional override for formatting stack traces
-         *
-         * @see https://v8.dev/docs/stack-trace-api#customizing-stack-traces
-         */
-        prepareStackTrace?: ((err: Error, stackTraces: NodeJS.CallSite[]) => any) | undefined;
-
-        stackTraceLimit: number;
-    }
-
-    /*-----------------------------------------------*
-    *                                               *
-    *                   GLOBAL                      *
-    *                                               *
-    ------------------------------------------------*/
-
     var global: typeof globalThis;
 
     var process: NodeJS.Process;
     var console: Console;
 
-    interface GCFunction {
```

Comment:
Has this been removed? `GCFunction` was previously visible.

Reply:
It was, in the global scope. It is now namespaced. I can't see any circumstance in which this would be consumed, and I can't see any usage on GitHub other than in DT forks.

Reply:
Hm, I swear I actually wrote code which used this type very recently... But I must have tossed it and I don't see it anymore.

Reply:
The new type is called `NodeJS.GC`; perhaps it should remain `GCFunction`? I'm not sure what's "right" here. I don't want to churn things too badly...

Reply:
(Other than this nit, seems fine to me overall)